src/wprof.c

```diff
@@ -368,8 +368,10 @@ static void print_exit_summary(struct worker_state *workers, int worker_cnt, str
 	int err;
 	u64 rb_handled_cnt = 0, rb_ignored_cnt = 0;
 	u64 rb_handled_sz = 0, rb_ignored_sz = 0;
-	struct wprof_stats stats_by_cpu[num_cpus] = {};
-	struct wprof_stats stats_by_rb[env.ringbuf_cnt] = {};
+	struct wprof_stats stats_by_cpu[num_cpus];
+	memset(stats_by_cpu, 0, num_cpus * sizeof(struct wprof_stats));
```

Comment:
please move memsets to after variable initialization block

also, just curious, using sizeof(stats_by_cpu) won't work with this kind of variable-sized on the stack arrays?